.travis.yml

```diff
@@ -9,7 +9,7 @@ php:
 services:
   - postgresql
 addons:
-  firefox: "latest"
+  firefox: "70.0.1"
```

Comment:
The existing tests run with firefox 72 e.g. https://travis-ci.org/owncloud/calendar/jobs/636625317?utm_medium=notification&utm_source=github_status

Why is Firefox 70 specified?

Reply:
It was failing sometimes in the firefox 71 and 72. I will change it back to the `latest` version. let's see what happens.

Reply:
@phi It fails in the latest versions https://travis-ci.org/owncloud/calendar/jobs/637799705?utm_medium=notification&utm_source=github_status

Reply:
OK - we can fix it to Firefox 70